### PR TITLE
lecture 7 - linear modelling

### PR DESCRIPTION
**Pull request by lcoome**

### CODE REVIEW

**Comment by QuLogic** [comment]
You'll need to add the new packages (car, psych, multcomp) to the `DESCRIPTION` file.

**Comment by lwjohnst86** [comment]
When in the `rcourse.Rproj` project, in the console type out:

```
usethis::use_package("car")
usethis::use_package("psych")
usethis::use_package("multcomp")
```

This will automatically add these packages to the `DESCRIPTION` file, in the correct location.

File: lecture07-linear-modelling.Rmd [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+equations ourselves is that we can focus on answering questions without worrying
+about whether we are doing the math or computation wrong. This is of course
+dependent on the type of research questions you may be interested in (e.g. for
+more theoretical questions, doing the math and computation yourself is probably
+a goal!) and on the type of data you are using/collecting. There is a lot of
+complexity that has already been taken care of in the available R packages and
+functions. For example, the function `lm` that we will use in this lesson uses the ordinary least squares (OLS) method, which is a common method for determining fit for linear models. That way, you can answer your research questions and not worry too much about the exact math involved and instead worry about the specifics of your field (e.g. Are you measuring the right thing? Are you collecting the right data? Are you asking the right questions? Is there an ecological or biological aspect you are missing in your analysis?)
+
+## Descriptive statistics
+### Frequencies with ``table()``:
```
Comment (lwjohnst86) [inline lecture07-linear-modelling.Rmd:63]: Btw, you only need one backtick ` around each word/function.

**Comment by QuLogic** [comment]
Done by #234.

**Pull request closed without merging**